### DISCUSSION TITLE
What Happened to the documentation ?!!

### DISCUSSION THREAD

**Discussion by omarelewa21** [livewire/livewire#5256] [1. Help]
The documentation server loads with no response.

**Comment by joshhanley** [comment]
@OmarElewa21 documentation is loading fine for me. Can you share some more details and a screenshot of the issue? Thanks!

**Reply by omarelewa21** [reply]
![Capture](https://user-images.githubusercontent.com/40321286/191455331-bde86ca9-6ff1-431d-8b2f-a075e69cfac9.PNG)

![Capture2](https://user-images.githubusercontent.com/40321286/191455709-2ac386fc-ee11-41c5-8f07-01142d5ed9f7.PNG)

**Reply by omarelewa21** [reply]
I have tried over different browsers but same problem !

**Reply by omarelewa21** [reply]
Everytime I switch off the VPN, the server doesn't give me a response. 
I am from Egypt.

**Comment by omarelewa21** [comment]
Ok, Now I tried with a VPN from USA and the documentation is visible !
But I wonder why ?
